### ISSUE TITLE
Update disabled state colors

### ISSUE THREAD

**Issue by hudajkhan** [sul-dlss/SearchWorks#5359]
Original ticket: In the "chat with librarian" section, there is both an icon and a link. When disabled, the icon is failing the color contrast test.  We need to see what color we can use for the disabled icon to pass the color contrast test.

We have other disabled states, such as the previous link for paging on search results and record view. 

Discussed with DLUX on 7/16/2025 and we decided it's not necessary to comply with contrast requirements based on this [criteria outlined by WCAG](https://www.w3.org/TR/WCAG21/#contrast-minimum):

> The visual presentation of [text](https://www.w3.org/TR/WCAG21/#dfn-text) and [images of text](https://www.w3.org/TR/WCAG21/#dfn-images-of-text) has a [contrast ratio](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) of at least 4.5:1, except for the following:
> 
> ... 
>     **Text or images of text that are part of an inactive [user interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components)**, that are [pure decoration](https://www.w3.org/TR/WCAG21/#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
> ... 

**TO DO: Color of disabled links should be Black-50 and they should be [labelled disabled](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-disabled).**

**Comment by hudajkhan** [comment]
Thanks @taylor-steve .
